lib/presenter/json_response.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+module Presenter
+  class JsonResponse
```

Comment:
Not required by task specification but added on my own accord to see how useful de-coupling presentation logic from the use case can be.